Update interfaces/cookie-store.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/3ef9f83/ed/idl/cookie-store.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/696958118

foolip commented:
This introduces a syntax error (missing semicolon) and will need a spec fix.

foolip commented on interfaces/cookie-store.idl:
```diff
@@ -40,33 +39,30 @@ enum CookieSameSite {
+  DOMTimeStamp? expires;
+  boolean secure;
+  CookieSameSite sameSite;
+}
```
Tests are failing because of the missing semicolon here.

foolip commented:
I've sent https://github.com/WICG/cookie-store/pull/145 to fix the problem.

autofoolip committed:
Update interfaces/cookie-store.idl

Source: https://github.com/tidoust/reffy-reports/blob/3ef9f83/ed/idl/cookie-store.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/696958118

foolip reviewed (approved):
https://wpt.fyi/results/cookie-store?run_id=596070001&run_id=575380002&run_id=586420002 looks OK now.
